i3_agenda/__init__.py

```diff
@@ -29,7 +29,8 @@
                    help='time for cache to be kept in minutes')
 parser.add_argument('--ids', '-i', type=str, default=[], nargs='+',
                     help='list of calendar ids to fetch, space separated. If none is specified all calendars will be fetched')
-
+parser.add_argument('--maxres', '-r', type=int, default=10, 
```

Comment:
It would be better to have a more specific help message to the user, maybe something like "max number of events to query Google's API for each of your calendars. Try increasing this if you notice that some events are missing or if you have lots of all day events."

Reply:
please review changes